src/components/SummaryList.tsx

```diff
@@ -51,23 +55,49 @@ export const SummaryList: React.FunctionComponent<SummaryListProps> = (
 
   const extraAttributes = Attributes.ariaAndData(props);
 
+  const anyRowHasActions = rows.some(
+    row => row.actions && row.actions.length > 0
+  );
+
   return (
     <dl
       id={id}
       className={classNames("govuk-summary-list lbh-summary-list", className)}
       {...extraAttributes}
     >
-      {rows.map(({ key, value }, index) => (
+      {rows.map((row, index) => (
         <div
           key={index}
           className="govuk-summary-list__row lbh-summary-list__row"
         >
           <dt className="govuk-summary-list__key lbh-summary-list__key">
-            {key}
+            {row.key}
           </dt>
           <dd className="govuk-summary-list__value lbh-summary-list__value">
-            {value}
+            {row.value}
           </dd>
+          {row.actions && row.actions.length > 0 ? (
+            <dd className="govuk-summary-list__actions">
+              {row.actions.length == 1 ? (
+                row.actions[0]
+              ) : (
+                <ul className="govuk-summary-list__actions-list">
+                  {row.actions.map((action, index) => {
+                    <li
+                      key={index}
+                      className="govuk-summary-list__actions-list-item"
+                    >
+                      {{ action }}
```

Comment:
```suggestion
                      { action }
```

Nunjucks slipping in there :).